pkg/config/api-server/catalog/config.go

```diff
@@ -2,15 +2,8 @@ package catalog
 
 // Not yet exposed via YAML and env vars on purpose
 type CatalogConfig struct {
-	ApiServer            ApiServerConfig               `yaml:"-"`
-	Bootstrap            BootstrapApiConfig            `yaml:"bootstrap"` // DEPRECATED: remove in next major version of Kuma
-	DataplaneToken       DataplaneTokenApiConfig       `yaml:"-"`         // DEPRECATED: remove in next major version of Kuma
-	MonitoringAssignment MonitoringAssignmentApiConfig `yaml:"monitoringAssignment"`
-	Sds                  SdsApiConfig                  `yaml:"sds"` // DEPRECATED: remove in next major version of Kuma
-}
-
-type ApiServerConfig struct {
-	Url string
+	Bootstrap      BootstrapApiConfig      `yaml:"bootstrap"` // DEPRECATED: remove in next major version of Kuma
```

Comment:
And this?

Reply:
If we remove it, Kuma DP 0.x will fail to connect to 1.0 CP